packages/cli/src/cog/cog.vrt.ts

```diff
@@ -102,7 +99,7 @@ export const CogVrt = {
 
         const inputTotal = job.source.files.length;
 
-        cutline.filterSourcesForName(name, job, sourceGeo);
+        cutline.filterSourcesForName(name, job);
```

Comment:
possibly unrelated to this PR
but do we really need to screw with the `job.json` could we maybe `const filesToUse = cutline.filter...`

Im not a huge fan of functions destroying a object that is commonly used in a bunch of places.

Reply:
I'll add to my TODO list

Reply:
Cool doesnt have to hold this PR up